### DISCUSSION TITLE
How to obtain screen resolution after setting the scaling ratio in Windows?

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#3054] [Q&A] [answered]
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/91530077/caad4c31-446c-47b2-97fc-f4c1950e5c7c)
As shown in the picture, my screen resolution is actually 2880 * 1800
But when I used flash. system. Capabilities. screenResolutionX to retrieve it, the result I got was 1440 (which is exactly half the scaling ratio). In fact, what I wanted to obtain was 2880
May I ask how can I know the current zoom setting value for Windows? Knowing the scaling ratio, I can calculate the resolution of the screen.

**Comment by ajwfrost** [comment, marked as answer]
Hi

It should be possible to get the scale factor from the updated `Screen` API (property added in 50.0):
https://airsdk.dev/reference/actionscript/3.0/flash/display/Screen.html#contentsScaleFactor

I think the `screenResolutionX` is the addressable resolution i.e. if you've got a display that has 2880x1800 pixels and yet have a scaling factor of 2x, then if we display an application at 1440x900 pixels, it will cover the whole screen...

thanks
